From 5428fca0cd542d4fb7beb781c1b44292aedb7915 Mon Sep 17 00:00:00 2001
From: "Masami Hiramatsu (Google)" <mhiramat@kernel.org>
Date: Mon, 20 Jul 2026 19:44:07 +0900
Subject: [PATCH 06144/15333] tracing/probes: Remove unused parameter from
 parse_probe_var_retval()

The parameter 'arg' in parse_probe_var_retval() is unused. Remove it
and update its caller accordingly.

Link: https://lore.kernel.org/all/178454424711.296567.15352344258939682112.stgit@devnote2/

Assisted-by: Antigravity:gemini-3.5-flash
Signed-off-by: Masami Hiramatsu (Google) <mhiramat@kernel.org>
---
 kernel/trace/trace_probe.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/trace/trace_probe.c b/kernel/trace/trace_probe.c
index 91ba4490937b..eb7be3725e85 100644
--- a/kernel/trace/trace_probe.c
+++ b/kernel/trace/trace_probe.c
@@ -1262,7 +1262,7 @@ NOKPROBE_SYMBOL(store_trace_entry_data)
 
 #define PARAM_MAX_STACK (THREAD_SIZE / sizeof(unsigned long))
 
-static int parse_probe_var_retval(char *orig_arg, char *arg,
+static int parse_probe_var_retval(char *orig_arg,
 				  struct fetch_insn **pcode,
 				  struct fetch_insn *end,
 				  struct traceprobe_parse_context *ctx)
@@ -1415,7 +1415,7 @@ static int parse_probe_vars(char *orig_arg, const struct fetch_type *t,
 		goto inval;
 
 	if (str_has_prefix(arg, "retval"))
-		return parse_probe_var_retval(orig_arg, arg, pcode, end, ctx);
+		return parse_probe_var_retval(orig_arg, pcode, end, ctx);
 
 	len = str_has_prefix(arg, "stack");
 	if (len)
-- 
2.55.0

